packages/wrangler/src/deploy/open-next.ts

```diff
@@ -0,0 +1,52 @@
+import { runCommand } from "../autoconfig/c3-vendor/command";
+import { getInstalledPackageVersion } from "../autoconfig/frameworks";
+import { logger } from "../logger";
+import { getPackageManager } from "../package-manager";
+
+/**
+ * Checks whether a project is an OpenNext one and in such case delegates the deployment to `@opennextjs/cloudflare`.
+ * If the project is not an OpenNext one, or the `OPEN_NEXT_DEPLOY` environment variable is set (meaning that `wrangler
+ * deploy` has been called from OpenNext itself), then no delegation happens.
+ *
+ * Why?
+ *  `wrangler deploy` shouldn't be run on an open-next project, since open-next has its own deploy command,
+ *  in the interest in having things "just work" we updated `wrangler deploy` to delegate to the open-next
+ *  deploy command, so that, if a developer runs `npx wrangler deploy` (e.g. like in Workers Builds) on an
+ *  open-next project that will work just fine.
+ *
+ * @param projectRoot The path to the project's root
+ * @returns true is the deployment has been delegated to open-next, false otherwise
```

Comment:
This is very verbose. Can we simplify to make it clearer what's going on here?

Reply:
I've simplified this comment: https://github.com/cloudflare/workers-sdk/pull/11301/commits/38859dc25c031be764e4408caf653e4c97af0be0

please let me know what you think